### PR TITLE
jobconfig fork for release-0.37

### PR DESCRIPTION
**Pull request by rmohr**
Use the correct config-forker to create unique names.

### CODE REVIEW

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.37.yaml [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 7h0m0s
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+    max_concurrency: 11
+    name: pull-kubevirt-e2e-k8s-1.20-0.37
```
Comment (dhiller) [inline github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.37.yaml:18]: not that it matters much, but shouldn't this be `pull-kubevirt-e2e-k8s-1.20-release-0.37`? Or is the length a problem?

**Review by dhiller — APPROVED** [review]
/lgtm
/approve
/hold

one minor question though

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.37.yaml (hunk 1 above)
Comment (rmohr) [reply github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.37.yaml:18]: Hm, it follows the usual `config-forker` convention of replacing `-master` with `-version`. Only the branches have the `release-` as prefix.

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.37.yaml [hunk 2]
```diff
@@ -0,0 +1,612 @@
...
+      grace_period: 30m0s
+      timeout: 4h0m0s
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+      rehearsal.allowed: "true"
+      sriov-pod: "true"
+    max_concurrency: 10
+    name: pull-kubevirt-e2e-kind-1.17-sriov-0.37
```
Comment (oshoval) [inline github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.37.yaml:204]: previous names were pull-kubevirt-e2e-kind-1.17-sriov-release-0.36

**Pull request merged**